includes/Modules/AdSense.php

```diff
@@ -421,6 +422,20 @@ protected function parse_data_response( Data_Request $data, $response ) {
 		return parent::parse_data_response( $data, $response );
 	}
 
+	/**
+	 * Checks for the state of an Account, whether close or not. 
+	 * 
+	 * @since 1.71.0
```

Comment:
This needs to say `n.e.x.t` based on our Site Kit-specific coding conventions:
```suggestion
	 * @since n.e.x.t
```